src/form/bf.jl

```diff
@@ -46,10 +46,56 @@ end
 
 "nothing to do, this model is symmetric"
```

Comment:
needs documentation of mathematical formulation here

Reply:
I have some writeup in latex, in overleaf,  not sure how to link it to the comments?

Reply:
You can drop in the latex, see for example https://github.com/lanl-ansi/PowerModelsDistribution.jl/blob/master/src/form/acp.jl#L345-L369